activities/communication/using-chat.md

```diff
@@ -0,0 +1,66 @@
+---
+type: guide
+---
+
+# How to use [chat.publiccode.net](https://chat.publiccode.net/)
+
+We use Mattermost, hosted at chat.publiccode.net for informal chat purposes.
+
+Mattermost lets us create multiple teams with different members. We only have 1 team right now: our staff team.
+
+## Staff team
+
+Discussions on chat.publiccode.net are:
+
+* timely
+* optional
+* secret
+
+If something must be known by everyone, please send an email.
```

Comment:
Anyone even? Or only if something must be known by exactly all staff?

Reply:
Good spot! I've fixed this now.